core/static/js/ga.js

```diff
@@ -0,0 +1 @@
+// override this file in your theme to add your own google analytics code
```

Comment:
(And by this I mean we should have general examples in the wiki or something, showing block overrides and block.super and all that, not a bunch of examples committed in the repository necessary, and definitely not in this branch)